content/aips/USDTETHV3-AIP.md

```diff
@@ -0,0 +1,92 @@
+---
+title: USDTethV3-AIP
```

Comment:
Better have a descriptive title as this is what will be shown on the governance UIs
```suggestion
Add USDT to Aave Ethereum V3 pool
```